appveyor.yml

```diff
@@ -29,7 +50,8 @@ build_script:
       "https://github.com/electron/electron"
   - gclient sync --with_branch_heads --with_tags
   - cd src
-  - gn gen out/Default "--args=import(\"//electron/build/args/%GN_CONFIG%.gn\") %GN_EXTRA_ARGS%"
+  - ps: $env:BUILD_CONFIG_PATH="//electron/build/args/%GN_CONFIG%.gn"
```

Comment:
This change isn't really needed.  I don't think that using `%` for environment variables in powershell works.

Reply:
I agree that it's not absolutely needed. But in the future we will get a cleaner diff when we decide to change that path.

>  I don't think that using % for environment variables in powershell works.

All Windows builds are green =)

Reply:
@alexeykuzmin yes, I saw the builds are green, but when I try that in powershell, I see the following:
```
PS C:\Users\jkleinsc> $env:GN_CONFIG="testing"

PS C:\Users\jkleinsc> $env:BUILD_CONFIG_PATH="//electron/build/args/%GN_CONFIG%.gn"

PS C:\Users\jkleinsc>echo $env:BUILD_CONFIG_PATH
//electron/build/args/%GN_CONFIG%.gn
```

Reply:
maybe appveyor is smart enough to convert or something.

Reply:
i don't understand how this works, why would the %% be substituted in powershell?